typing_tester.py

```diff
@@ -0,0 +1,173 @@
+import curses
+from curses import wrapper
+import time
+import requests
+
+def get_random_word():
+    """Get random word with pronunciation from RapidApi"""
+
+    url = "https://random-words-api-umber.vercel.app/word"
+
+    headers = {
+        "X-RapidAPI-Key": "f934e406b8msh16f9dee78137dfcp1fab91jsn560d9925327d",
```

Comment:
Figure out whats the `.env` file, why do we use it and how can we use it in python. Make use of .env here